### ISSUE TITLE
Pie chart for supply/conversion technologies

### ISSUE THREAD

**Issue by daren-thomas** [architecture-building-systems/hive#215]
("capacity_hp", "capacity_chp", "capacity_boiler")

**Comment by christophwaibel** [comment]
note that I'm trying to follow a flexible approach in the Results class:

```
        /// <summary>
        /// Names for each supply technology, e.g. "Boiler", "CHP", ...
        /// </summary>
        public string[] SupplyNames { get; private set; }
        /// <summary>
        /// Matrix defining the suitability of each technology per carrier type.
        /// Rows: Technologies
        /// Columns: [0]: Electricity, [1]: Heating, [2]: Cooling
        /// </summary>
        public bool[,] SupplyTypes { get; private set; }
        /// <summary>
        /// Capacity per technology. Unit is defined in 'SupplyCapUnits
        /// </summary>
        public double[] SupplyCapacities { get; private set; }
```

**Comment by amraladdin** [comment]
Group w/ systems that provide electricity, cogeneration, heat only... etc.

@amraladdin Be creative... Doesnt' have to be a Pie chart

**Comment by amraladdin** [comment]
Hey @christophwaibel do you have a list of the technologies we have? I remember there was an issue about that.

**Comment by christophwaibel** [comment]
@amraladdin so, I would aim for a flexible approach, where it reads any technology from `string [] SupplyNames`:

![image](https://user-images.githubusercontent.com/21927657/84916312-36547180-b0be-11ea-9c92-b69d3ab49143.png)

Let's say, `SupplyNames = {"boiler", "ASHP", "PV", "battery", "chiller"}`, then it plots these technologies.
the units would come from `string [] SupplyCapUnits` and the capacity from `double [] SupplyCapacities` and the suitability (heating, cooling, elec, co-gen) from the matrix `bool[,] SupplyTypes`

does that make sense?

**Comment by amraladdin** [comment]
Exactly what I was working on :)

**Comment by daren-thomas** [comment]
@ChrisZenhub why multiple lists of strings etc? what about a `SupplySystemDefinition` object? to group the stuff together that belongs together?

**Comment by christophwaibel** [comment]
> `SupplySystemDefinition`

@daren-thomas good point, that would make a lot of sense to feed in all the objects that I'm moving around and processing within the core also into the `Visualizer`. 

I already have a class `Hive.IO.EnergySystem.Generation`. I am currently extending it such that it contains timeseries as well (which are populated within Hive.Core). 

Ok, so I would need a bit more time until I restructured the dataflow and data structure from Hive Input into Hive.Core into Hive Visualizer.

If you already want to work on this graph ,for now you could consider that you read following class:

```
public class EnergySystem.Conversion
{
	/// <summary>
	/// Technology name
	/// </summary>
	public string Name { get; protected set; }
	/// <summary>
	/// Indicating whether this technology produces electricity
	/// </summary>
	public bool IsElectric { get; protected set; }
	/// <summary>
	/// Indicating whether this technology produces heat
	/// </summary>
	public bool IsHeating { get; protected set; }
	/// <summary>
	/// Indicating whether this technology produces cooling
	/// </summary>
	public bool IsCooling { get; protected set; }
	/// <summary>
	/// Capacity of technology. Unit is defined in 'CapacityUnit'
	/// </summary>
	public double Capacity { get; protected set; }
	/// <summary>
	/// Unit of technology capacity (e.g. "kW", or "sqm", etc.)
	/// </summary>
	public string CapacityUnit { get; protected set; }
	
	public Conversion(){ }
}
```

**Comment by amraladdin** [comment]
Update: I'm removing myself as Assignee from this issue since there's no more work to be done from my side.

**Comment by amraladdin** [comment]
Update: We still need to decide how this plot should look like.

**Comment by christophwaibel** [comment]
just decide it then ;-) you're the designer

**Comment by amraladdin** [comment]
That was a reminder for me that this issue was not finished yet, but thanks for the comment :D

Here we go:
![image](https://user-images.githubusercontent.com/16256867/86532211-5c548100-bec8-11ea-89a7-97ae2575d4ba.png)

The colors here correspond to the colors in the monthly demand issue #209 
Here are the values again for your convenience:
DHW
192, 0, 0
Space heating
255, 0, 0
Space cooling
0, 176, 240
Electricity
255, 217, 102

**Comment by daren-thomas** [comment]
@christophwaibel how do I find this data?
